### DISCUSSION TITLE
Welcome to stackline Discussions!

### DISCUSSION THREAD

**Discussion by AdamWagner** [AdamWagner/stackline#58] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by molleweide** [comment]
Hey man this looks really cool. I forked the project. I just started out using yabai. How can i unstack my windows?

**Reply by AdamWagner** [reply]
Hi @molleweide!

Check @johnallen3d 's script to unstack windows here:

https://github.com/AdamWagner/stackline/issues/70

You can also unstack windows by "warping" a stacked window somewhere else. The only caveat to this is that you need at least two nodes on the active space … otherwise there isn't anywhere for a window to be warped ;-)
